tests/mock_web_api_server.py

```diff
@@ -22,9 +23,9 @@ def is_valid_token(self):
     def is_valid_user_token(self):
         return "Authorization" in self.headers and str(self.headers["Authorization"]).startswith("Bearer xoxp-")
 
-    def set_common_headers(self):
+    def set_common_headers(self, content_length: int = 0):
         self.send_header("content-type", "application/json;charset=utf-8")
-        self.send_header("connection", "close")
+        self.send_header("Content-Length", str(content_length))
```

Comment:
nit: As the above "content-type" works well, consistently using "content-length" (all lower case) would look sightly nicer to me
```suggestion
        self.send_header("content-length", str(content_length))
```